src/Fhir.Anonymizer.AzureDataFactoryPipeline/src/DataFactoryCustomActivity.cs

```diff
@@ -120,43 +136,43 @@ private async Task AnonymizeBlobInJsonFormatAsync(BlobClient inputBlobClient, Bl
             }
         }
 
-        private async Task AnonymizeBlobInNdJsonFormatAsync(BlobClient inputBlobClient, BlobClient outputBlobClient, string blobName)
+        private async Task AnonymizeBlobInNdJsonFormatAsync(BlobClient inputBlobClient, BlockBlobClient outputBlobClient, string blobName)
```

Comment:
We also need to cover cross region support on json files, even though we do not support parallel processing on json file yet.

Reply:
Sure, we ran 10 rounds of synthea files json format of cross region scenario. Thanks for the suggestion.